aws-xray-recorder-sdk-core/src/test/java/com/amazonaws/xray/contexts/LambdaSegmentContextTest.java

```diff
@@ -49,6 +50,8 @@ class LambdaSegmentContextTest {
 
     private static final String MALFORMED_TRACE_HEADER =
         ";;Root=1-57ff426a-80c11c39b0c928905eb0828d;;Parent=1234abcd1234abcd;;;Sampled=1;;;";
+    private static final String MALFORMED_TRACE_HEADER_2 =
```

Comment:
Please add a test case "Root=1-5759e988-bd862e3fe1be46a994272711;Lineage=10:1234abcd:3", naming it as `Root Lambda passthrough`